ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/JsonRpcMethodsFactory.java

```diff
@@ -248,7 +249,8 @@
           new MinerStart(miningCoordinator),
           new MinerStop(miningCoordinator),
           minerSetCoinbase,
-          new MinerSetEtherbase(minerSetCoinbase));
+          new MinerSetEtherbase(minerSetCoinbase),
+          new TxPoolPendingTransactions(transactionPool.getPendingTransactions()));
```

Comment:
Not sure if this is the right place for this. I placed it in the mining section for now.